pinot-spi/src/main/java/org/apache/pinot/spi/data/DateTimeFormatSpec.java

```diff
@@ -47,28 +49,69 @@ public class DateTimeFormatSpec {
   public static final int MIN_FORMAT_TOKENS = 3;
   public static final int MAX_FORMAT_TOKENS = 4;
 
+  public static final int FORMAT_TIMEFORMAT_POSITION_PIPE = 0;
+  public static final int MIN_FORMAT_TOKENS_PIPE = 1;
+  public static final int MAX_FORMAT_TOKENS_PIPE = 3;
+
+  //Applicable for SDF|<timeFormat>(|<timezone>)
```

Comment:
(minor) Also add a space between the comment and `//`
```suggestion
  // Applicable for SIMPLE_DATE_FORMAT|<timeFormat>(|<timezone>)
```